src/components/Header/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { StateContext } from '../../store/State';
+
+export const Header: React.FC = () => {
+  const { todos } = useContext(StateContext);
+
+  return (
+    <header className="todoapp__header">
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button
+        type="button"
+        className={classNames('todoapp__toggle-all', {
+          active: todos.some(({ completed }) => !completed),
```

Comment:
move logic into variable that use here 
```
const isActiveButton = todos.some(({ completed }) => !completed);
```
```suggestion
          active: isActiveButton,
```